### DISCUSSION TITLE
Creating SecRules dynamically

### DISCUSSION THREAD

**Discussion by yasir-ss** [corazawaf/coraza#918] [Q&A]
Is it possible to create SecRules dynamically, such as through an API? From what I understand, the rules are specified in a file, and you have to restart Coraza in order to load the new config.

**Comment by jcchavezs** [comment]
Not supported but default but intel developed something called ruleset server which I believe they would be interested in release to work with coraza aswell cc @ns-sundar

**Reply by yasir-ss** [reply]
Can you provide more details on the rule set server?

**Reply by jcchavezs** [reply]
https://github.com/intel/modsecurity-wasm-filter/tree/main/ruleserver

**Reply by faf-xff** [reply]
> Not supported but default but intel developed something called ruleset server which I believe they would be interested in release to work with coraza aswell cc @ns-sundar

Why can't we provide an API? It would make usage much more convenient.
